MySQLPlugin.cpp

```diff
@@ -223,7 +223,6 @@ void CMySQLPlugin::OnScript_Affected_Rows()
 
     int idx = getHandleIndexForScriptArg(0);
     checkConnection(idx);
-    checkQuery(idx);
```

Comment:
Can't agree. You can't just remove it from here. Simply because you can't get affected rows count without query.
There must be other solution: right after query store errno value in class for connection. If result is NULL and errno is 0 then everything is ok.